selfdrive/car/gm/interface.py

```diff
@@ -212,7 +212,14 @@ def _update(self, c):
     if ret.cruiseState.standstill:
       events.add(EventName.resumeRequired)
     if ret.vEgo < self.CP.minSteerSpeed:
-      events.add(car.CarEvent.EventName.belowSteerSpeed)
+      events.add(EventName.belowSteerSpeed)
+
+    if self.CP.networkLocation == NetworkLocation.fwdCamera:
+      # The ECM has a higher brake pressed threshold than the camera, causing an
+      # ACC fault when you engage at a stop with your foot partially on the brake
+      # TODO: use ECM's standstill threshold
+      if ret.vEgoRaw <= 0.5 and ret.brake <= 20:
+        events.add(EventName.depressBrakePedal)
```

Comment:
Can't use brakePressed since it includes regen paddle which doesn't count as brake pressed to ECM at a standstill